### PR TITLE
Add systemd collector

### PR DESCRIPTION
**Pull request by gmlexx**
Send 1 if configured systemd service status is running and 0 otherwise

### CODE REVIEW

File: src/collectors/systemd/systemd-collector.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+            manager = Manager()
+
+            for service in [s.strip() for s in self.config['services']]:
+
+                if not service:
+                    continue
+
+                try:
+                    unit = manager.get_unit('%s.service' % service)
+                    self.publish("%s.running" % service, 1
```
Comment (bmhatfield) [inline src/collectors/systemd/systemd-collector.py]: Are there more statistics about systemd services we can send here, beyond just whether or not a service is enabled?

Comment (gmlexx) [inline src/collectors/systemd/systemd-collector.py]: There are many other properties that we can access (look at http://www.freedesktop.org/wiki/Software/systemd/dbus/ Unit Objects properties), but there are not so useful as is service actually running or not.

File: src/collectors/elasticsearch/elasticsearch.py [hunk 2]
```diff
@@ -152,7 +152,7 @@ def _index_metrics(self, metrics, prefix, index):
         # publish all 'total' and 'time_in_millis' stats
         self._copy_two_level(
             metrics, prefix, index,
-            lambda key: key.endswith('total') or key.endswith('time_in_millis'))
+            lambda key: key.endswith('total') or key.endswith('time_in_millis') or key.endswith('in_bytes') or key.endswith('count'))
```
Comment (eMerzh) [inline src/collectors/elasticsearch/elasticsearch.py]: Is it really meant to be in the same PR ? i don't see the relation....?!

Comment (gmlexx) [inline src/collectors/elasticsearch/elasticsearch.py]: Surely not. PR was from master. One non-related commit has been added.

**Pull request closed without merging**